.gitignore

```diff
@@ -4,7 +4,7 @@ logs
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-/apps/win32/src/reports/
+/apps/win32/reports/
```

Comment:
Will this need to be wildcarded later?

Reply:
What do you mean by wildcarded?

Once I add more platforms, I believe I will change this to "**/reports/"